web/yo/app/scripts/controllers/gene.js

```diff
@@ -1833,7 +1836,7 @@ angular.module('oncokbApp')
 
                 DatabaseConnector.updateGene(params, function(result) {
                     $scope.docStatus.savedGene = true;
-                    changeLastUpdate();
+                    mainUtils.updateLastSavedToDB();
```

Comment:
Just curious, shouldn't these be included in database connector?

Reply:
If you are talking about where to call mainUtils.updateLastSavedToDB() after saving changes in Review mode, then it needs to be called in gene.js. Because various APIs are fired depending on what type of update it is, and it is complicated to track all kinds of APIs sent out. But  modelUpdate() in gene.js is always called after APIs calls are finished, that is the correct place to mark last update detail. 
If you are talking about the Update Gene specifically, mainUtils.updateLastSavedToDB() can be called either in gene.js or database connector, it doesn't matter very much. But just to keep consistent with other saving function, I put it in gene.js.